stdlib/private/SwiftPrivateThreadExtras/SwiftPrivateThreadExtras.swift

```diff
@@ -96,7 +96,7 @@ public func _stdlib_thread_create_block<Argument, Result>(
   if threadID == 0 {
     return (errno, nil)
   } else {
-    return (0, UnsafeMutablePointer<ThreadHandle>(&threadID).pointee)
```

Comment:
\*blink\* This wasn't even doing the right thing under the old code; it's entirely possible the temporary would get overwritten before the `pointee` gets read.

Reply:
If something's UB with C pointers, it's UB with Swift pointers too.

Reply:
Oddly enough, didn't run into any issues; guess we were getting lucky.  At least it is fixed now.